tools/render-test/slang-support.cpp

```diff
@@ -251,11 +251,13 @@ static SlangResult _compileProgramImpl(
 
     String source(request.source.dataBegin, request.source.dataEnd);
     ComPtr<slang::IBlob> diagnostics;
-    ComPtr<slang::IModule> module(slangSession->loadModuleFromSourceString(
-        "main",
-        request.source.path,
-        source.getBuffer(),
-        diagnostics.writeRef()));
+    ComPtr<slang::IModule> module(
```

Comment:
module returned by slangSession->loadModuleXX is always owned by session and will always be destroyed when session is destroyed. The external reference count on `IModule` is simply ignored. You shouldn't need this change here.

Reply:
I would like to understand why using an ordinary ComPtr would cause leaks. Why can't this be
```
auto module = slangSession->loadModuleFromString();
```
?